astro/src/content/docs/apis/_tenant-response-body-base.mdx

```diff
@@ -353,6 +361,10 @@ import JSON from 'src/components/JSON.astro';
     When enabled, all refresh tokens will be revoked when a user enables multi-factor authentication for the first time. This policy will not be applied when adding subsequent multi-factor methods to the user.
   </APIField>
 
+  <APIField name={props.base_field_name + '.jwtConfiguration.refreshTokenRevocationPolicy.onOneTimeTokenReuse'} type="Boolean" since="1.55.0">
```

Comment:
```suggestion
  <APIField name={props.base_field_name + '.jwtConfiguration.refreshTokenRevocationPolicy.onOneTimeTokenReuse'} type="Boolean" since="1.55.1">
```